File: beecrowd/1020 - Idade em Dias.rs
use std::io;

fn read_i32() -> i32 {
    let mut linha = String::new();

    io::stdin()
        .read_line(&mut linha)
        .expect("Falha ao tentar ler a linha");

    let valor: i32 = linha.trim()
                        .parse::<i32>()
                        .expect("Falha ao interpretar como inteiro");

    return valor;
}

fn main() {

    let mut tempo = read_i32();

    let anos = tempo / 365;

    tempo %= 365;

    let meses = tempo / 30;
    let dias = tempo % 30;

    println!("{} ano(s)\n{} mes(es)\n{} dia(s)", anos, meses, dias);
}

File: beecrowd/1020 - Idade em Dias.go
package main

import "fmt"

func main() {
	var dias int32

	const anos2Dias = 365
	const meses2Dias = 30

	fmt.Scan(&dias)

	anos := dias / anos2Dias
	dias %= anos2Dias
	meses := dias / meses2Dias
	dias %= meses2Dias

	fmt.Printf("%d ano(s)\n", anos)
	fmt.Printf("%d mes(es)\n", meses)
	fmt.Printf("%d dia(s)\n", dias)
}
